Automattic/jetpack
# CRM: Fix `pending` Woo order status is not a paid order

Fixes: https://github.com/Automattic/zero-bs-crm/issues/3087

## Proposed changes:
This PR fixes a small mistake introduced in this merged PR: https://github.com/Automattic/jetpack/pull/30557. The WooCommerce order status, `pending`, it's not considered an order is paid, simply changing it to `completed` it'll fix it.

Also, this PR added a link to the WooCommerce order statuses doc, in the WooSync setting page: `/wp-admin/admin.php?page=zerobscrm-plugin-settings&tab=woosync`

https://woocommerce.com/document/managing-orders/

![image](https://github.com/Automattic/jetpack/assets/9832440/3a613f01-9bbe-4aef-b21c-b48ccc216cfb)


### Other information:


## Jetpack product discussion
N/A

## Does this pull request change what data or activity we track or use?
N/A

## Testing instructions:
- Go to the WooSync setting page and you will see the correct mapping for the Order status, `Processing` and `Completed` are mapped to the `Paid` invoice status.
- Check the link to the WooCommerce order status doc
